src/pages/home/ReportScreen.tsx

```diff
@@ -333,6 +333,14 @@ function ReportScreen({
     }
 
     const transactionThreadReportID = useMemo(() => ReportActionsUtils.getOneTransactionThreadReportID(report.reportID, reportActions ?? []), [report.reportID, reportActions]);
+
+    useEffect(() => {
+        if (!transactionThreadReportID || !route.params.reportActionID) {
```

Comment:
We also need to add condition to check if the current report is one transaction thread too. Otherwise it will cause https://github.com/Expensify/App/issues/44588. More details in [this comment](https://github.com/Expensify/App/issues/44588#issuecomment-2196179504).